data/src/main/java/ca/bc/gov/data/local/entity/TestRecordEntity.kt

```diff
@@ -45,5 +46,7 @@ data class TestRecordEntity(
     @ColumnInfo(name = "result_desc")
     val resultDescription: String,
     @ColumnInfo(name = "result_link")
-    val resultLink: String
+    val resultLink: String,
+    @ColumnInfo(name = "data_source", defaultValue = "")
```

Comment:
For TestRecordEntity default_source is not required. This extra column is sufficient in TestResultEntity.